TheraLang.BLL/Services/AuthenticationService.cs

```diff
@@ -50,19 +50,25 @@ public async Task<string> Authenticate(User user)
         }
 
         public async Task<AuthUser> GetAuthUserAsync()
+        {
+            return await TryGetAuthUser() ?? throw new Exception($"Error while getting user id from token");
+        }
+
+        public async Task<AuthUser> TryGetAuthUser()
```

Comment:
it's either not good naming or we should refactor this method to match `bool TryGetValue(id, our value)` signature
https://ayoungdeveloper.com/post/2017-03-28-using-the-tryget-pattern-in-csharp/

Reply:
Cannot use out param in an async method so we decided to return value if it's valid and null if not